### PR TITLE
fix(setup): make setup really read the file content

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: mergify_cli/__init__.py [hunk 1]
```diff
@@ -105,8 +105,7 @@ def get_slug(url: str) -> tuple[str, str]:
         path = parsed.path[1:].rstrip("/")
 
     user, repo = path.split("/", 1)
-    if repo.endswith(".git"):
-        repo = repo[:-4]
+    repo = repo.removesuffix(".git")
```
Comment (jd) [inline mergify_cli/__init__.py:108]: just do this in a different pr instead of  all pr lol

Comment (sileht) [reply mergify_cli/__init__.py:108]: lol, It's not me, I guess my editor fix this crap alone 🤣
